src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -4,6 +4,7 @@
  * Add some custom fields that could be only in Excavator
  * Do not remove no-args constructor
  */
+
 public class Excavator extends Machine {
```

Comment:
Don't begin class implementation with an empty line. According to the checklist, the class definition should not be preceded by an empty line.